Python/Product/Debugger/Guids.cs

```diff
@@ -14,7 +14,7 @@
 
 using System;
 
-namespace Microsoft.PythonTools.DkmDebugger {
+namespace Microsoft.PythonTools.Debugger {

```

Comment:
These are public APIs (unless they're new in 2.2, which I don't think they are), so we can't rename the namespace unfortunately.